### PR TITLE
Minor FAQ update

### PR DESCRIPTION
**Pull request by lucasmogu**
Some improvements to "What an ideal progress report looks like?"

### CODE REVIEW

**Review by Arvin21M — APPROVED** [review]
This looks good to me! 🚀

File: data/pages/faq-grantees.mdx [hunk 1]
```diff
@@ -105,11 +105,11 @@ evaluate progress.
 
 ### What does an ideal progress report look like?
 
-An ideal progress report is:
-
 - Written in Markdown format
 - Between 1 and 3 pages
-- Enriched with links to pull requests, commits, and other relevant work produced.
+- Enriched with the most relevant links to pull requests, commits, and other work produced.
+- Has a brief summary in each section before diving into links in order to help us contextualize the links provided.
```
Comment (BoltTouring) [inline data/pages/faq-grantees.mdx:111]: - Has a brief summary in each section before providing additional detail (links, PRs, etc)

Comment (lucasmogu) [reply data/pages/faq-grantees.mdx]: [Commit here](https://github.com/OpenSats/website/pull/335/commits/ee29b8e56c46b82699468bfafb36855192f4aecb)

**Review by dergigi — APPROVED** [review]
Looks good to me ✅

**Pull request merged**